tools/azure-sdk-tools/tests/test_pyproject_interactions.py

```diff
@@ -0,0 +1,81 @@
+import os, tempfile, shutil
+
+
+import pytest
+
+from ci_tools.parsing import update_build_config, get_build_config, get_config_setting
+
+integration_folder = os.path.join(os.path.dirname(__file__), "integration")
+pyproject_folder = os.path.join(integration_folder, "scenarios", "sample_pyprojects")
+pyproject_file = os.path.join(integration_folder, "scenarios", "sample_pyprojects", "pyproject.toml")
+
+
+@pytest.mark.parametrize(
+    "target",
+    [
+        # check is true
+        pyproject_file,
+        pyproject_folder,
+        os.path.join(pyproject_folder, "setup.py"),
+    ],
+)
+def test_pyproject_parse(target):
+    config = get_build_config(target)
+
+    assert config == {"mypy": True, "type_check_samples": True, "verifytypes": True, "pyright": False}
+
+
+def test_pyproject_file_parse():
```

Comment:
This test looks like it's redundant after the update to `test_pyproject_parse`

Reply:
Agreed.